client/bindings/java/lib/src/main/java/org/iota/apis/NodeCoreApi.java

```diff
@@ -159,6 +159,15 @@ public Block getIncludedBlock(TransactionId transactionId) throws ClientExceptio
         return new Block(responsePayload);
     }
 
+    public Block getIncludedBlockMetadata(TransactionId transactionId) throws ClientException {
+        JsonObject o = new JsonObject();
+        o.addProperty("transactionId", transactionId.toString());
+
+        JsonObject responsePayload = (JsonObject) nativeApi.sendCommand(new ClientCommand("getIncludedBlockMetadata", o));
+
+        return new Block(responsePayload);
```

Comment:
Hmm it should return a `BlockMetadata` object, shouldn't it?
But I guess we don't have this yet there?

So basically what you can do is clone the `Block` object and name it to `BlockMetadata` and return that here.
Also make sure you have an import statement then when you use it here.

Else you could return `JsonObject` simply for now, and I will do this later.